src/components/fields/ArrayField.js

```diff
@@ -117,8 +224,10 @@ class ArrayField extends Component {
 
   onReorderClick = (index, newIndex) => {
     return (event) => {
-      event.preventDefault();
-      event.target.blur();
+      if (event) {
+        event.preventDefault();
+        event.target.blur();
+      }
```

Comment:
I added this so that you could programmatically call the `onDropIndexClick()()` without a `MouseEvent` argument.